app.js

```diff
@@ -65,8 +72,17 @@ app.use(expressWinston.errorLogger({
   ],
 }));
 
-const routers = require('./api/routers');
+const limiter = new RateLimit({
```

Comment:
One thought I have is we could add this to a config, something like `config.rateLimitting`. Then you could configure production like this, and do this for the default:

```
{
  max: 0,
  delayAfter: 0,
  delayMs: 0
}
```

That should enable rate limiting in prod, but disable it in dev and test. But we could still get and idea of how the app behaves when requests are passed through the middleware in the test and dev environment.

Thoughts?